reference/atomic/atomic_flag_test_explicit.md

```diff
@@ -47,10 +47,10 @@ int main()
   std::cout << std::boolalpha;
 
   std::atomic_flag x = ATOMIC_FLAG_INIT;
-  std::cout << std::atomic_flag_test(&x, std::memory_order::acquire) << std::endl;
+  std::cout << std::atomic_flag_test_explicit(&x, std::memory_order::acquire) << std::endl;
 
   std::atomic_flag_test_and_set(&x, std::memory_order::release);
-  std::cout << std::atomic_flag_test(&x, std::memory_order::acquire) << std::endl;
+  std::cout << std::atomic_flag_test_explicit(&x, std::memory_order::acquire) << std::endl;
 }
 ```
 * std::atomic_flag_test[color ff0000]
```

Comment:
```suggestion
* std::atomic_flag_test_explicit[color ff0000]
```

Reply:
Thanks for your suggestion. https://github.com/cpprefjp/site/pull/791/commits/f5062a55daec52663416fecbe7485b2826474eaf